src/components/Forms/AddSupplier/AddSupplier.js

```diff
@@ -15,113 +14,219 @@ import { useNavigation } from "contexts/NavigationContext";
+      // console.log("Please fill out required field");
+      openSnackbar("Please fill out required field");
     }
   };
 
   const snackbar = () => {
-    openSnackbar("Successfully added a new supplier!");
+    openSnackbar("Successfully added new supplier.");
     goBack();
   };
 
   return (
-    <div className={styles["container"]}>
-      <LeftChevron />
-      <form
-        className={styles["form"]}
-        onSubmit={(event) => handleSubmit(event)}
-      >
-        <div className={styles["form-header"]}>
-          <ApartmentIcon style={{ fontSize: "40px" }} />
-          <b>&nbsp; ADD SUPPLIER</b>
-        </div>
-        <div className={styles["form-body"]}>
-          <Grid container spacing={3}>
-            <Grid item xs={12} sm={6}>
-              <InputField
-                required
-                label={"Supplier Name"}
-                value={name}
-                setValue={setName}
-              />
-            </Grid>
-            <Grid item xs={12} sm={6}>
-              <InputField
-                label={"Contact Name"}
-                value={contact}
-                setValue={setContact}
-              />
-            </Grid>
-            <Grid item xs={12} sm={6}>
-              <InputField
-                label={"Email Address"}
-                value={email}
-                setValue={setEmail}
-              />
-            </Grid>
-            <Grid item xs={12} sm={6}>
-              <InputField
-                label={"Phone Number"}
-                value={phone}
-                setValue={setPhone}
-              />
-            </Grid>
-            <Grid item xs={12}>
-              <InputField
-                label={"Address"}
-                value={address}
-                setValue={setAddress}
-              />
+    <>
+      <div className={styles["add-suppplier"]}>
+        {/* create supplier modal */}
+        <Modal
+          openModal={openModal}
+          setOpenModal={setOpenModal}
+          dialogTitle="ARE YOU SURE?"
+          dialogContentText={`You are about to create ${name}`}
+          onClose={() => setOpenModal(false)}
+        >
+          <Button onClick={() => setOpenModal(false)} size="large">
+            Cancel
+          </Button>
+          <Button
+            onClick={handleSubmit}
+            size="large"
+            variant="contained"
+            // disabled={submitting}
+          >
+            {/* {submitting ? <CircularProgress /> : "REMOVE"} */}
+            CREATE
+          </Button>
+        </Modal>
+        <LeftChevron />
+        <h1 className={styles["header"]}>NEW SUPPLIER</h1>
+
+        <form>
+          <Grid container justify="space-evenly">
+            <Grid item xs={12} sm={5} container id="leftCol">
+              <div className={styles["supplier-container"]}>
+                <Grid item xs={12} spacing={1} container>
+                  <Grid item container justify="space-between">
+                    <InputField
```

Comment:
hey leels can you just update this to have the Supplier name at the top. Then maybe username can be 'website username' so it's not confusing and be placed right below website field. Thanks!

Reply:
okay, gotcha!